<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loaded Character</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>791</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	color:rgb(255,255,255);
	border-style:none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_cima">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	color: rgb(255,255,255);
	background-color: rgb(64, 128, 191);
	border-color:none;
	border-style:none;
	border-radius:15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Personagem Encontrado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_central">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>601</width>
       <height>231</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	color: rgb(255,255,255);
	background-color: rgb(64, 128, 191);
	border-color:none;
	border-style:none;
	border-radius:10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="charactername">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color:rgb(94, 226, 255);
	border-style:none;
	border-radius:0px;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="OK">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(74, 167, 219);
	border-style:none;
	border-radius:10px;
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color:rgb(94, 226, 255);
}</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="shortcut">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NO">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(74, 167, 219);
	border-style:none;
	border-radius:10px;
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color:rgb(94, 226, 255);
}</string>
      </property>
      <property name="text">
       <string>NO</string>
      </property>
      <property name="shortcut">
       <string>N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>581</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Seu personagem da versão beta foi encontrado, deseja carregá-lo agora?</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
